assembly/UInt/UInt128.ts

```diff
@@ -34,6 +34,25 @@ export class UInt128 implements Codec {
             this.bitLength = 1 + valueInBytes.length;
         }
     }
+    /**
+     * @description Non-static constructor method used to populate defined properties of the model
+     * @param bytes SCALE encoded bytes
+     * @param index index to start decoding the bytes from
+     */
+    populateFromBytes(bytes: u8[], index: i32 = 0): void{
```

Comment:
Can we extract this logic into _ private functions for internal use in order to remove the code duplication?